lib/models/device.coffee

```diff
@@ -562,6 +562,36 @@ getDeviceModel = (deps, opts) ->
 	exports.isOnline = (uuidOrId, callback) ->
 		exports.get(uuidOrId, $select: 'is_online').get('is_online').asCallback(callback)
 
+	###*
+	# @summary Check if a device is active
+	# @name isActive
+	# @public
+	# @function
+	# @memberof resin.models.device
+	#
+	# @param {String|Number} uuidOrId - device uuid (string) or id (number)
+	# @fulfil {Boolean} - is device active
+	# @returns {Promise}
+	#
+	# @example
+	# resin.models.device.isActive('7cf02a6').then(function(isActive) {
+	# 	console.log('Is device isActive?', isActive);
+	# });
+	#
+	# @example
+	# resin.models.device.isActive(123).then(function(isActive) {
+	# 	console.log('Is device active?', isActive);
+	# });
+	#
+	# @example
+	# resin.models.device.isActive('7cf02a6', function(error, isActive) {
+	# 	if (error) throw error;
+	# 	console.log('Is device online?', isActive);
+	# });
+	###
+	exports.isActive = (uuidOrId, callback) ->
+		exports.get(uuidOrId, $select: 'is_active').get('is_active').asCallback(callback)
```

Comment:
What's the use case for this method? We generally don't want to add a separate method in the SDK for each field, unless there's a really good reason somebody would need an easy way to specifically look up this one field.

In the UI for example, I imagine we'd load this as part of the `devices.getAll()/device.get(id)` call when loading the content for the page, not with a separate request for each device involved.